frontend/src/App.svelte

```diff
@@ -109,9 +113,13 @@
 
   function handleShortcut(e) {
     // Don't handle keypresses from within text inputs.
+    // Disable shortcuts in views with text inputs, otherwise misclicking outside of a text area may
+    // cause users to accidentally run shortcuts.
     if (
       ["input", "textarea"].includes(e.target?.tagName.toLocaleLowerCase()) ||
-      e.target.isContentEditable
+      e.target.isContentEditable ||
+      modifierView === ComponentConfig ||
+      modifierView === FindReplaceView
```

Comment:
Probably makes sense for this to just check that the `modifierView` variable is set at all.

```suggestion
      modifierView
```